tools/rtrclient.c

```diff
@@ -230,6 +247,45 @@ static bool is_readable_file(const char *str)
 	return is_file(str);
 }
 
+#ifdef RTRLIB_HAVE_LIBSSH
+static bool is_utf8(const char *str)
+{
+	size_t len = strlen(str);
+
+	for (size_t i = 0; i <= len; ++i) {
+		// check if current byte is a single utf char
+		if ((str[i] & UTF8_ONE_BYTE_MASK) == UTF8_ONE_BYTE_PREFIX) {
+			continue;
+
+			// check if current byte is the start of a two byte utf8 char and validate subsequent bytes
```

Comment:
minor: indention of comment not aligned like comment above? Same for next 2 `else if`.

Reply:
fixed